### PR TITLE
Separate all emojis by comas in channel list

### PR DESCRIPTION
**Pull request by CoBC**
Also replace "question" by ❓ for question mode

### CODE REVIEW

**Comment by CoBC** [comment]
@bear101 I'm still searching emojis for "Administrator", "Channel operator" and "Away"
Do you have any idea, or do you think we should keep text for those?

**Comment by bear101** [comment]
Administrator: 🥷

Channel Operator: 👮‍♂️

Away: 😴

**Comment by bear101** [comment]
Notice there's a female police as well 👮‍♀️ 👮‍♂️ 👮

**Comment by bear101** [comment]
Woman administrator could be 🦸‍♀️

File: Client/qtTeamTalk/channelstree.cpp [hunk 1]
```diff
@@ -1218,14 +1218,14 @@ void ChannelsTree::updateUserItem(QTreeWidgetItem* item)
...
         else if ((user.nStatusMode & STATUSMODE_GENDER_MASK) == STATUSMODE_MALE)
-            itemtext += (_Q(user.szStatusMsg).size() ? " 👨" : ", 👨");
+            itemtext += ", 👨";
         if(user.uUserType & USERTYPE_ADMIN)
-            itemtext += " (" + ((user.nStatusMode & STATUSMODE_FEMALE)?tr("Administrator", "For female"):tr("Administrator", "For male and neutral")) + ")";
+            itemtext += user.nStatusMode & STATUSMODE_FEMALE?" (🦸‍♀️)":"(🥷)";
 
         if (TT_IsChannelOperator(ttInst, user.nUserID, user.nChannelID))
-            itemtext += " (" + ((user.nStatusMode & STATUSMODE_FEMALE)?tr("Channel operator", "For female"):tr("Channel operator", "For male and neutral")) + ")";
+            itemtext += user.nStatusMode & STATUSMODE_FEMALE?"(👮‍♀️)":"(👮‍♂️)";
```
Comment (bear101) [inline Client/qtTeamTalk/channelstree.cpp:1228]: is there a space missing here? There's a space on line 1225

**Pull request merged**